### DISCUSSION TITLE
Seeking Help (would pay) for Creating SvelteKit Turborepo

### DISCUSSION THREAD

**Discussion by git-no** [vercel/turborepo#1016] [Help] [answered]
I want to build up a working Sveltekit Turborepo and looking for help (instructions, answers) to get this done.
Should include tailwindcss, markdown (mdsvex) and centralized lint, prettier, jsconfig ... configuration and an example of a simple component. 
I would pay for successfully creating such turborepo repository.

I want to use JSDoc not Typescript, because it is simple handling.
I know there is at GitHub a Sveltekit Turborepo with Typescript.

**Comment by oneezy** [comment, marked as answer]
https://github.com/oneezy/monorepo

**Comment by mehulkar** [comment]
We have a Svelte example in this repo as well that may be helpful, but @oneezy 's response looks like it includes sveltekit! https://github.com/vercel/turborepo/tree/main/examples/with-svelte
